### PR TITLE
fixed contribution count bug

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: mod/wikirate/set/all/contributions.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 def update_direct_contribution_count
-  return unless direct_contribution_count_card
+  return unless respond_to? :direct_contribution_count
  
   new_contr_count = intrusive_family_acts.count
   Card::Auth.as_bot do
     if direct_contribution_count_card.new_card?
-      direct_contribution_count_card.update_attributes!(:db_content => new_contr_count.to_s)
+      direct_contribution_count_card.update_attributes!(:content => new_contr_count.to_s)
```
Comment (ethn) [inline mod/wikirate/set/all/contributions.rb:7]: back to this?

I still think we should be able to get this done without running all the callbacks for each of these cards...

Comment (xithan) [inline mod/wikirate/set/all/contributions.rb:7]: I didn't find something like "update_column" for a new entry. I tried to turn off callbacks with `skip_callbacks( [:approve, :store, :extend] )` but it didn't work

Comment (ethn) [inline mod/wikirate/set/all/contributions.rb:7]: even though it's ugly, I'm tempted to use update_attributes for new cards and update_column for existing ones.  Perhaps we could add our own method to do that.

Comment (ethn) [inline mod/wikirate/set/all/contributions.rb:7]: something like:

```
def quick_update_content content
  if new?
    update_attributes! :content=>content
  else
    update_column :db_content, content
  end
end
```

**Pull request merged**